engine/buildkit/executor.go

```diff
@@ -1,4 +1,4 @@
-package main
+package buildkit
```

Comment:
Could we add a note to this file from where it's pulled from in buildkit? Just thinking that like the git source/etc, we'll need to make sure that we port changes/fixes from upstream here.

Out of curiosity, do you see a path here to making some updates upstream so that we could re-use more parts of the runc executor specifically? It feels like we want some generic "hooks" before+after the runc execution, and that might let us get the required cacerts/ftp_proxying stuff.

Not a thing to necessarily work on right now, but I have some future work planned where I do want to remove our custom git source by allowing it to interface neatly with upstream, so I might also include this - if you think it's potentially at all possible.

Reply:
Yeah, fair enough, all very legitimate :laughing: Agreed with all the above points!

Reply:
Added comment with that link